src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ CHANGELOG
  * Made `framework.session.handler_id` accept a DSN
  * Marked the `RouterDataCollector` class as `@final`.
  * [BC Break] The `framework.messenger.buses.<name>.middleware` config key is not deeply merged anymore.
+ * The `TemplateController` now accepts context argument like described in the official documentation
```

Comment:
I think the last part if the sentence is not necessary (rm `like described in the official documentation`)